langgraph/src/pregel/types.ts

```diff
@@ -0,0 +1,38 @@
+import { Runnable, RunnableConfig } from "@langchain/core/runnables";
+
+export interface PregelTaskDescription {
+  name: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  input: any;
+}
+
+export interface PregelExecutableTask {
+  name: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  input: any;
+  proc: Runnable;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  writes: Array<[string, any]>; // TODO: Array type may need to be changed
```

Comment:
The Python implementation uses `collections.deque` and the implementation of `Pregel` depends on the `deque` APIs. I need to figure out the appropriate type later.

Reply:
we use deque in python for thread safety. in js fine to use array